Retrieve link name from link when using link asset (#401)
semantic-release/gitlab: README.md

| Property | Description                                                                                                 | Default                              |
| -------- | ----------------------------------------------------------------------------------------------------------- | ------------------------------------ |
| `path`   | **Required**, unless `url` is set. A [glob](https://github.com/isaacs/node-glob#glob-primer) to identify the files to upload.    | -                                    |

> | `url`    | Alternative to setting `path` this provides the ability to add links to releases, e.g. URLs to container images. The url must be unique within the release.                           | -                                    |

Reviewer: ```suggestion
| `url`    | Alternative to setting `path` this provides the ability to add links to releases, e.g. URLs to container images. Must be unique within the release.                           | -                                    |
```

> | `label`  | Short description of the file displayed on the GitLab release. Can be dynamically adjusted with [Lodash template](https://lodash.com/docs#template). Allows same variables as [`successComment`](#successComment). Ignored if `path` matches more than one file. The Label must be unique within the release when `url` is set. | File name extracted from the `path`. File name extracted from `url` |

Reviewer: I'd simplify this a bit:

```suggestion
| `label`  | Short description of the file displayed on the GitLab release. Can be dynamically adjusted with [Lodash template](https://lodash.com/docs#template). Allows same variables as [`successComment`](#successComment). Ignored if `path` matches more than one file. Must be unique within the release. | File name extracted from `path` or `url` |
```
